lib/redmine_tags/patches/wiki_controller_patch.rb

```diff
@@ -1,3 +1,21 @@
+# This file is a part of redmine_tags
```

Comment:
I wonder, should this license header be included in all files?
What I understood from my google searches is that a LICENSE file in the root folder is enough (at least for FOSS).
Also some projects (like [elixir](https://github.com/elixir-lang/elixir)) do not include it.
I also read stories about needing to update the Copyright year with a huge commit at the beginning of the year.
I am not saying to remove them from all files, but maybe not include them in new ones? @ixti what do you think?